corehq/apps/app_manager/templates/app_manager/v1/form_designer.html

```diff
@@ -159,3 +162,47 @@
 {% endblock %}
 
 {% block column_style %}hq-flush-content{% endblock %}
+
+{% block modals %}
+    {{ block.super }}
+    {% with slug="vellum_beta" name="New Form Builder" %}
+    {% initial_page_data "toggle_item" toggle_item %}
+    {% initial_page_data "toggle_url" toggle_url %}
+    {% if not request|toggle_enabled:'VELLUM_BETA' %}
+        <!-- This will appear on page load, so don't use any animation (normally controlled by .fade) -->
+        {% blocktrans %}
```

Comment:
Did you intend to have this entire block translated? If you did, what do you think of moving the `{% blocktrans %}` inside `<div class="modal-content">` so there's less HTML to be processed by the translators?